erpnext/accounts/doctype/accounting_dimension_filter/test_accounting_dimension_filter.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and Contributors
+# See license.txt
+from __future__ import unicode_literals
+
+import frappe
+import unittest
+from erpnext.accounts.doctype.sales_invoice.test_sales_invoice import create_sales_invoice
+from erpnext.accounts.doctype.accounting_dimension.test_accounting_dimension import create_dimension, disable_dimension
+from erpnext.exceptions import InvalidAccountDimension, MandatoryDimension
```

Comment:
Consistent error class naming
```suggestion
from erpnext.exceptions import InvalidAccountDimensionError, MandatoryAccountDimensionError
```